orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/AtlasQueueMonitor.kt

```diff
@@ -71,6 +84,23 @@ class AtlasQueueMonitor
     _lastState.set(queue.readState())
   }
 
+  @Scheduled(fixedDelayString = "\${queue.zombie.check.frequency:3600000}")
+  fun checkForZombies() {
+    val startedAt = clock.instant()
+    repository.retrieve(PIPELINE)
+      .mergeWith(repository.retrieve(ORCHESTRATION))
```

Comment:
I'd really like to make methods to specifically fetch running pipelines. Wouldn't make much difference for Redis since we'd still likely filter in memory but it would be much zippier with SQL.

Reply:
Yes, 💯 . Just a `retrieve(ExecutionType, ExecutionCriteria)` method would have a lot of utility.

Reply:
Ok, gimme a minute & I'll make that happen.